Core/GDCore/Extensions/Metadata/InstructionMetadata.cpp

```diff
@@ -118,6 +120,8 @@ InstructionMetadata& InstructionMetadata::UseStandardOperatorParameters(
     }
   } else {
     AddParameter("operator", _("Modification's sign"), type);
+    // The type "string" is not forced because it's declined in several subtype
+    // (see ParameterMetadata).
     AddParameter(type == "number" ? "expression" : type, _("Value"));

```

Comment:
I'm thinking to use this to handle types like `color`. I'm not sure this is intended though.
`ParameterMetadata::IsExpression` could be used here to make it more explicit and robust. Unless, it should only be "string" but it seems fine to do the change.